pallets/avn-transaction-payment/src/tests/set_known_sender_tests.rs

```diff
@@ -33,6 +34,18 @@ mod set_known_senders {
                     account_1,
                     config,
                 ));
+
+                assert!(event_emitted(&mock::RuntimeEvent::AvnTransactionPayment(
+                    crate::Event::<TestRuntime>::KnownSenderAdded {
+                        known_sender: account_1,
+                        adjustment: FeeAdjustmentConfig::<TestRuntime>::PercentageFee(
+                            PercentageFeeConfig {
+                                percentage: 10,
```

Comment:
what are these constants in the configs? How are they related to the rest of the tests, or does it not matter?